### ISSUE TITLE
Mise en conformité de la page d'accueil BetaGouv

### ISSUE THREAD

**Issue by freesteph** [betagouv/beta.gouv.fr#9885]
BetaGouv doit respecter les normes d'accessibilité du RGAA (https://www.numerique.gouv.fr/publications/rgaa-accessibilite/#contenu).

Cette issue est en cours pour identifier tous les problèmes de la page d'accueil de BetaGouv.

**Comment by freesteph** [comment]
- la hiérarchie des titres n'est pas respectée : on passe de `<h2>` à `<h4>`:

<img width="336" alt="Capture d’écran 2022-05-04 à 12 10 38" src="https://user-images.githubusercontent.com/107635/166662838-b4f6565b-25ca-4729-9b5f-ddfab39dae34.png">

**Comment by freesteph** [comment]
Certaines images n'ont pas de texte alternatif, ou un `alt=""` (ou `aria-hidden=true`) pour indiquer qu'elles sont purement décoratives

**Comment by freesteph** [comment]
Certaines emphases sont indiquées purement par la couleur : 

![Capture d’écran 2022-05-04 à 12 28 38](https://user-images.githubusercontent.com/107635/166664885-ba2e109d-636e-48b5-ab66-307fdf755d50.png)

**Comment by freesteph** [comment]
Certaines abréviations n'ont pas de sens:

```
<abbr title="mesr.md">MESRI</abbr>
```

**Comment by freesteph** [comment]
> Certaines abréviations n'ont pas de sens:
> 
> ```
> <abbr title="mesr.md">MESRI</abbr>
> ```

Au temps pour moi c'était une erreur en local et pas du tout un problème en production.

**Comment by revolunet** [comment]
@astranchet c'est toujours d'actu ?

**Comment by astranchet** [comment]
Oui, je clotûre (il y a d'autres problèmes d'accessibilité identifiés, il faut qu'on voie ensemble où je peux les renseigner)
